sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -181,3 +182,13 @@ def test_connectivity(seed=36):
     assert_equal(_graph_is_connected(graph), True)
     assert_equal(_graph_is_connected(csr_matrix(graph)), True)
     assert_equal(_graph_is_connected(csc_matrix(graph)), True)
+
+
+def test_spectral_embedding_deterministic():
+    """Test that Spectral Embedding is deterministic"""
```

Comment:
@ragv : This is what I have done, please tell if it would be enough ?
1. Finding the indexes of maximum absolute value in all the vectors.
2. Flipping the vector signs by running ```_eigen_sign_flip```
3. Finding the indexes of maximum value in all the vectors.
4. Asserting that indexes in step 1 and step 3 are same by using ```assert_array_equal```